db/migrate/20240718095751_drop_user_table.rb

```diff
@@ -0,0 +1,14 @@
+class DropUserTable < ActiveRecord::Migration[7.1]
```

Comment:
question: why we have created 2 migration for user deletion?

Reply:
in the first migration, there was a mistake by me, so it did not delete user table ,the mistake was ,i put down methed change table